README.md

```diff
@@ -9,6 +9,10 @@ Date.first_day_of_month_for(dat,day)
 - dat can be, date object / date string
 - day can be, case insensitive string of weekdays / below mentioned numbers representing weekdays
 
+Date.wednesday_of_year(week, year)
```

Comment:
I think this can be dynamically done for all days. For wednesday_of_year, if I remember correctly, I need to get date for xth wednesday in a year. This can be conveniently provided for all days of week.

I can think about multiple ways of doing that:
1. use method_missing (ugly and I will not prefer)
2. dynamically define all methods using a collection and iterate through each day of week.

Let me know, I can add the second option if you like.